### PR TITLE
update #161 UI for Profile Image and User Info

### PR DESCRIPTION
**Pull request by rkalra247**

### CODE REVIEW

File: @types/user.d.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export type User = {
+  userName: string
```
Comment (songz) [inline @types/user.d.ts:2]: `username` is used everywhere, not `userName`

File: components/ProfileImageInfo.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+import '../scss/profileImageInfo.scss'
+
+type ProfileImageInfoProps = {
+  user: User
+}
+
+const ProfileImageInfo: React.FC<ProfileImageInfoProps> = props => {
+  return (
+    <div className="card shadow-sm">
+      <div className="profile_image_placeholder text-center">
```
Comment (songz) [inline components/ProfileImageInfo.tsx:13]: `profile_image_container`

File: components/ProfileImageInfo.tsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+const ProfileImageInfo: React.FC<ProfileImageInfoProps> = props => {
+  return (
+    <div className="card shadow-sm">
+      <div className="profile_image_container text-center">
+        <UserInfoImage user={props.user} />
+        <h2 className="profile_image_full_name text-center">
+          {`${props.user.firstName} ${props.user.lastName}`}
+        </h2>
+        <h4 className="profile_image_username text-center text-muted">
+          {'@' + props.user.username}
```
Comment (Wolfy64) [inline components/ProfileImageInfo.tsx:19]: You use string interpolation above, why not use here and stay consistent
``` `@  ${props.user.username}` ```

File: components/SubmissionCard.tsx [hunk 4]
```diff
@@ -1,23 +1,26 @@
 import React from 'react'
 import Button from './Button'
+import UserInfoImage from './UserInfoImage'
+import { User } from '../@types/user'
+import '../scss/submissionCard.scss'
 
 type Props = {
   userId: string
```
Comment (Wolfy64) [inline components/SubmissionCard.tsx:8]: I know we did not do on other types but should we not use `userId: ID!` instead of `string`

Comment (songz) [reply components/SubmissionCard.tsx:8]: what is `ID` used for?

Comment (Wolfy64) [reply components/SubmissionCard.tsx:8]: Nothing, I mixed up Type and GraphQL^^

File: components/ProfileImageInfo.tsx [hunk 5]
```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import UserInfoImage from './UserInfoImage'
+import { User } from '../@types/user'
+import '../scss/profileImageInfo.scss'
+
+type ProfileImageInfoProps = {
+  user: User
+}
+
+const ProfileImageInfo: React.FC<ProfileImageInfoProps> = props => {
```
Comment (coltonehrman) [inline components/ProfileImageInfo.tsx:10]: `({ user })` instead of `props`

File: components/SubmissionCard.tsx [hunk 6]
```diff
@@ -1,23 +1,26 @@
...
-  initial: string
   time: string
   challengeTitle: string
   reviewUrl: string
+  user: User
 }
 
 const SubmissionCard: React.FC<Props> = props => {
```
Comment (coltonehrman) [inline components/SubmissionCard.tsx:15]: `({ user, ... })` instead of `props`

**Pull request merged**